packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -199,13 +190,34 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
                         ],
                       ),
                     ),
+                  Console(),
+                ],
+              ),
             ),
           ],
         ),
       ],
     );
   }
 
+  Container buildScriptTitle(ThemeData theme) {
+    return Container(
+      decoration: BoxDecoration(
```

Comment:
This Container is used in multiple places. Can we add a helper method in common.dart `debuggerSectionTitle` and use it here, in the console, and in the debugger pane headers if those are the same?

Reply:
I consolidated the logic here. The debugger pane headers need a bit more logic, so the code is still separate, but we did reduce some duplication.